### ISSUE TITLE
Issue with trading shiny zapdos

### ISSUE THREAD

**Issue by BATShatner** [FoglyOgly/Meowth#229]
In the trading channel,  if a users tries to trade with or for shiny zapdos,  it strips shiny and just puts zapdos

**Comment by FoglyOgly** [comment]
Might be you're running an old commit that doesn't have shiny Zapdos. Repo
might also be behind a bit. Check meowth/exts/pokemon.py and look for the
shiny list.

**Comment by BATShatner** [comment]
Thanks... but I did a pull a couple of Nights ago when scragly fixed the unown issue and that is fixed.
I did try adding to the shiny_list array (as it was not there) -- but it didn't work for me and I assumed it might be more than just adding it to that list.

EDIT :To clarify --  I backed up pokemon.py -- I added 145 to the shiny_list,  and did a !reload pokemon  and !reload_json for good measure.. and had the same issue,  so I  killed/reloaded the bot completely and had the same issue and copied back over my edit for future git/pulls.   If that should work.. maybe i did something wrong -- I will try again after work.
